src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -59,6 +376,40 @@ def __init__(self, sc, *args):
             self.ml = sc._jvm.org.apache.sysml.api.MLContext(sc._jsc, monitorPerformance, setForcedSparkExecType)
         except Py4JError:
             traceback.print_exc()
+            
+    def start(self):
+        try:
+            prevGlobalMLContext = MLContext.globalMLContext
+        except AttributeError:
+            pass
+        self.numTabs = 0
+        self.sqlCtx = SQLContext(self.sc)
+        MLContext.globalMLContext = self
```

Comment:
Is this assignment necessary to do? What happens if we replace every usage of `MLContext.globalMLContext` with `self`